PrizeDraw/MainWindow.xaml.cs

```diff
@@ -127,61 +135,73 @@ async void MainWindow_KeyDown(object sender, KeyEventArgs e)
             {
                 case Key.Space:
                 case Key.Next:
-                {
-                    _viewModel.StartNextMode();
-                    break;
-                }
+                    {
+                        _viewModel.StartNextMode();
+                        break;
+                    }
                 case Key.Escape:
                 case Key.B:
-                {
-                    _viewModel.Restart();
-
-                    while (Canvas.Children.Count > 0)
                     {
-                        Canvas.Children.Remove(Canvas.Children[0]);
-                    }
+                        if (_viewModel?.SelectedTile != null)
+                        {
+                            _viewModel.SelectedTile.IsWinner = false;
+                            _viewModel.SelectedTile.LoadNoshowImage(imageFolder);
+                        }
+
+                        // Can we replace the tile with the VieUserControl
```

Comment:
Sounds like a reasonable enhancement. My motivation for the PR really is to prevent us from drawing people more than once per event (whether a winner or a no-show). I think perhaps add a new issue for that to keep things progressing?

Reply:
Looking at the changes the I've just committed and other comments, the image location is no longer an issue. Just the `ObservableCollection` to look at.